### DISCUSSION TITLE
Cloudflare Pages + Vite not building correctly when deploying to CF

### DISCUSSION THREAD

**Discussion by joaoguidev** [remix-run/remix#9099] [Q&A] [answered]
Hello.

My Vite project works fine in development but when I deploy to Cloudflare It does not load some Chuncks, I'm using Tailwind and csrf from Remix utils:

![image](https://github.com/remix-run/remix/assets/63625334/b992e7d8-5a54-4a9c-aba0-8a0f56f07a9d)

Here is my setting on Cloudflare:

![image](https://github.com/remix-run/remix/assets/63625334/c139af60-8a07-45a0-a693-e6c3556f795c)

**Comment by iakshay** [comment, marked as answer]
The output directory should be `./build/client`
